### ISSUE TITLE
Package request: suyu

### ISSUE THREAD

**Issue by jakehamilton** [NixOS/nixpkgs#293811]
**Project description**

Suyu (pronounced "Sue You" *wink wink*) is written in C++ with portability in mind, and we actively maintain builds for Windows, Linux and Android.

**Metadata**

* homepage URL: https://gitlab.com/suyu-emu/suyu
* source URL: https://gitlab.com/suyu-emu/suyu
* license: GPLv3
* platforms: linux

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by dotlambda** [comment]
See https://github.com/NixOS/nixpkgs/pull/293312#issuecomment-1980212857.

**Comment by Aleksanaa** [comment]
Let's wait until they've made another release.

**Comment by jakehamilton** [comment]
I agree, I think it's best to wait. I created the package request so we had an issue for it when the time was right rather than discussion in separate threads everywhere. At least we can track it here 👍

**Comment by Noodlez1232** [comment]
I made a package for it, and wrapped it in a flake, but I don't know if I should open a PR for it to be added.

**Comment by KonstantinDjairo** [comment]
how about a guix package ?

**Comment by Aleksanaa** [comment]
> I made a package for it, and wrapped it in a flake, but I don't know if I should open a PR for it to be added.

If you want to add packages freely while not making yet another flake, you can go to http://nur.nix-community.org/ . Packages added to Nixpkgs are reviewed for format, functionality, etc., and may not be merged in the time you expect.

**Comment by tengkuizdihar** [comment]
the first release is here! https://gitlab.com/suyu-emu/suyu/-/releases/v0.0.2-master

**Comment by jakehamilton** [comment]
The project has moved from GitLab to a Forgejo instance: https://git.suyu.dev/suyu/suyu/src/branch/dev

**Comment by AndersonTorres** [comment]
> The project has moved from GitLab to a Forgejo instance: https://git.suyu.dev/suyu/suyu/src/branch/dev

They still link gitlab. The project is still messy, I would wait a bit more.

**Comment by tengkuizdihar** [comment]
I agree, I think we should let the project marinate a bit and see if they
have a stable home first. That git instance is self hosted after all.

**Comment by Aleksanaa** [comment]
Anyone still remember this? Suyu is v0.0.3 now

**Comment by mksafavi** [comment]
> Anyone still remember this? Suyu is v0.0.3 now

I haven't followed the yuzu forks recently, but I think Suyu got shutdown a few days after that release.

**Comment by PaulGrandperrin** [comment]
The repository and bug tracker looks very much alive: https://git.suyu.dev/suyu/suyu/activity/monthly

**Comment by mksafavi** [comment]
> The repository and bug tracker looks very much alive: https://git.suyu.dev/suyu/suyu/activity/monthly

Great 👍
Last time I checked, I remember their gitlab account and discord server were taken down. They made an statement that they decided to stop the development. They probably changed their mind.
Hope it survives   ✌️

**Comment by nyabinary** [comment]
There is torzu too.

**Comment by LDprg** [comment]
Suyu still developed. I would really like to see it available on nixpkgs or at least at nur.

**Comment by AndersonTorres** [comment]
The most serious problem around Suyu is that no big forge wants to deal with that company that manufactures playing cards.

**Comment by tengkuizdihar** [comment]
@LDprg found this on nur https://github.com/nix-community/nur-combined/blob/master/repos/chigyutendies/pkgs/suyu/dev.nix#L169

**Comment by ynejmidev** [comment]
they already provide a flake at https://git.suyu.dev/suyu/nix-flake

**Comment by Redhawk18** [comment]
I believe it's worth a try at least, the project seems to be starting to getting the wheels rolling now. But keep in mind it will take them time to actually learn and figure out that emulator, but it seems like a very alive project now. I could try to package if people would support it.

**Comment by tengkuizdihar** [comment]
@AndersonTorres GRIFFIIIIIIITH


thanks for the PR!
